### ISSUE TITLE
Please add Qwen-audio

### ISSUE THREAD

**Issue by zimuoo** [ollama/ollama#3471]

**Comment by tikeoewoew** [comment]
Please add Qwen-VL:
https://huggingface.co/Qwen/Qwen-VL

**Comment by iplayfast** [comment]
I think you meant https://huggingface.co/Qwen/Qwen-Audio and That's a really neat idea!

**Comment by zimuoo** [comment]
Yes,thanks!发自我的 iPhone在 2024年4月4日，00:01，Chris Bruner ***@***.***> 写道：﻿
I think you meant https://huggingface.co/Qwen/Qwen-Audio and That's a really neat idea!

**Comment by testmana2** [comment]
hello, thank you for your amazing working! and I wonder how it's going? the newest qwen2-audio model was here: https://huggingface.co/Qwen/Qwen2-Audio-7B-Instruct

**Comment by goetzc** [comment]
Having audio models support in Ollama would be great, as currently not many nice tools support this, AFAIK only [LocalAI](https://localai.io) does.

Additionally to Qwen2-Audio, I would like to see support for OpenAI's [Whisper](https://github.com/openai/whisper), which is also open source.

**Comment by philipag** [comment]
The new NVIDIA Parakeet and related models are very good.

**Comment by reneleonhardt** [comment]
Unfortunately this project doesn't seem to have any interest in those amazing new local models 😞

NVIDIA's New Open-Source Tool to Revolutionize Multilingual Speech-to-Text Experience
https://www.communeify.com/en/blog/parakeet-tdt-0-6b-v3-nvidia-opensource-multilingual-stt/
https://huggingface.co/nvidia/parakeet-tdt-0.6b-v3
